### PR TITLE
[Misc][UI] Default Case for Challenge Rules

### PR DESCRIPTION
**Pull request by frutescens**
## What are the changes the user will see?
None.

## Why am I making these changes?
Ease of Development. 

## What are the changes from a developer perspective?
Developers no longer have to worry about editing run-info-ui-handler when creating a new challenge

### Screenshots/Videos
image

## How to test the changes?
Start a run with Fresh Start on. 

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I considered writing automated tests for the issue?
- [x] If I have text, did I make it translatable and add a key in the English locale file(s)?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Review by DayKev — APPROVED** [review]

File: src/ui/run-info-ui-handler.ts [hunk 1]
```diff
@@ -587,13 +587,13 @@ export default class RunInfoUiHandler extends UiHandler {
           const typeText = typeTextColor + typeShadowColor + i18next.t(`pokemonInfo:Type.${typeRule}`)! + "[/color]" + "[/shadow]";
           rules.push(typeText);
           break;
-        case Challenges.FRESH_START:
-          rules.push(i18next.t("challenges:freshStart.name"));
-          break;
         case Challenges.INVERSE_BATTLE:
-          //
           rules.push(i18next.t("challenges:inverseBattle.shortName"));
```
Comment (Tempo-anon) [inline src/ui/run-info-ui-handler.ts:591]: How come inverse still has its own case despite being a binary on/off challenge?

Comment (frutescens) [reply src/ui/run-info-ui-handler.ts:591]: b/c Inverse is listed as 'Inverse Battle'

Comment (Tempo-anon) [reply src/ui/run-info-ui-handler.ts:591]: wonder if that can be changed so this isn't a unique case but w/e

Comment (frutescens) [reply src/ui/run-info-ui-handler.ts:591]: yeah, it'll be confusing on the challenge selection screen otherwise.

**Pull request merged**